client/src/pages/EpisodeShowPage/WorkspaceRelations.ts

```diff
@@ -34,19 +32,12 @@ const WORKSPACE = "WORKSPACE";
 const SUBWORKSPACE = "SUBWORKSPACE";
 const ROOT_WORKSPACE = "ROOT_WORKSPACE";
 
-const isUserInExperiment = getIsUserInExperimentFromQueryParams(
-  window.location.search,
-);
-
 const RelationTypeAttributes = [
   {
     name: WorkspaceRelationTypes.WorkspaceQuestion,
     source: WORKSPACE,
     blockType: QUESTION,
-    permission:
-      Auth.isAdmin() && !isUserInExperiment
-        ? Permissions.Editable
-        : Permissions.ReadOnly,
```

Comment:
Looks like this removes the ability of admins to edit the Question. Was this your intention?

Reply:
Yeah. Sometimes admins participate in experiments with their admin accounts (maybe we should discourage this, but it's a separate point). And there's now an edge case situation (when an honest expert admin is viewing the "original judge workspace" when deciding whether to concede to malicious or play out subtree) where we want admins in this situation to see the workspace question in the same way a non-admin would.

Reply:
@zjmiller FWIW I think it'd be fine to tell admins not to participate in experiments with their admin accounts

I think it'd be helpful if you wrote up a short guide for people who have an admin account to:
- participate in an experiment
- make changes to the tree via the UI
- see workspaces just like the participant in the workspace would have seen it (to the extent that's possible)

and it'd be fine if for some of these the best thing is to use a non-admin account